Header header

int16 object_class
float32 detection_score

int16 xmin
int16 xmax
int16 ymin
int16 ymax

float32 variance_xmin
float32 variance_xmax
float32 variance_ymin
float32 variance_ymax